release/models/policy/openconfig-policy-types.yang

```diff
@@ -283,4 +289,13 @@ module openconfig-policy-types {
 
       10.244.136.79/32.";
   }
+
+  identity OTHER {
```

Comment:
I also don't think we should add this -- again, is there a reason not to just add the protocols that are not listed here?

Reply:
I can recall a discussion we had many years ago on a related topic. :-)

I'm not sure it is realistic to extend the list of identities to be a superset of ALL the "install" protocol types across ALL vendors, especially when you consider that some vendors may have protocol types that are proprietary and not tied to standardized protocols. And even if you could come up with a list that is 99% complete now the OC model will always be in catch-up mode as new protocols get implemented.

If you are against 'other' then it should be documented that an implementation is free to provide no value for this leaf if the installed type is not expressible in OC.

Reply:
I think we are ok in OC in general to rapidly update identities and enums in OC.   PR's can be merged pretty quickly for these kinds of changes (2 weeks or even less for obvious, documented identity definitions.  An example is OC Ethernet PMD types). This is suitable for protocols that are supported by 2 or more vendors.

For truly vendor proprietary protocols, we could prescribe a specific model structure for vendors to augment an OC model.  In this case, it could be as simple as creating a 'vendor-protocol-identity' leaf which refers to identity provided by a vendor in a separate, vendor specific yang module.    (I would not want to mix/modify a list of OC identities with vendor identities due to confusion and collisions between vendors).  Examples of this are the [pipeline-counters ](https://github.com/openconfig/public/blob/master/doc/vendor_counter_guide.md) model.

Reply:
Hi @dplore , sure let me resubmit the PR with some obvious missing identities that I know are supported by 2+ vendors, and also add a vendor container that could be an attachment point for vendor-specific identities that come from augmentation.

Reply:
This is an identity -- it can be extended by vendors through augments for their proprietary protocols, you can simply use the `install-protocol-type` identity as the `base`.

Reply:
Vendors could indeed add additional identities with an augment.  I suppose there is some remote chance that there could be a conflict where the same identity is used by two different vendors but has different meanings.  This seems unlikely, but might be very confusing if it were to occur.   Using a "vendor" container is slightly more complex, but would guarantee this won't happen.